package.json

```diff
@@ -17,87 +17,87 @@
     "test:unit": "vue-cli-service test:unit"
   },
   "dependencies": {
-    "@lumino/algorithm": "^1.3.2",
-    "@lumino/datagrid": "^0.3.1",
-    "@lumino/default-theme": "^0.1.8",
-    "@lumino/widgets": "^1.9.3",
-    "apollo-boost": "^0.4.4",
-    "apollo-utilities": "^1.3.2",
+    "@lumino/algorithm": "^1.3.3",
+    "@lumino/datagrid": "^0.12.0",
+    "@lumino/default-theme": "^0.4.4",
+    "@lumino/widgets": "^1.13.4",
+    "apollo-boost": "^0.4.9",
+    "apollo-utilities": "^1.3.4",
     "axios-fetch": "^1.1.0",
-    "cytoscape": "^3.9.0",
+    "cytoscape": "^3.15.2",
     "cytoscape-dagre": "^2.2.2",
-    "cytoscape-expand-collapse": "^3.1.2",
-    "cytoscape-node-html-label": "^1.1.5",
+    "cytoscape-expand-collapse": "^4.0.0",
+    "cytoscape-node-html-label": "^1.2.0",
```

Comment:
I'm updating the Graph dependencies, assuming they won't require us to change anything in the Graph component or view :+1: if they demand further work, I will remove these changes to simplify the PR, since we may choose a different library for handling Graphs in Cylc UI.

Reply:
I think we may want to remove the graph view next release, pending @oliver-sanders massive re-do. Probably not good to have an barely-usable graph view with wrong look-and-feel alongside our nice (and now performant, post deltas) tree view. (n-distance windows might save graph view performance, but still have the other problems...).

Reply:
At least removing dependencies and code is easier. Just need to remember to remove not only `cytoscape-` dependencies, but also the `tippy` I think. Would be nice if there were dependency groups in NPM, or if at least we could add comments to `package.json` (JSON limitation)